lang/syn/src/idl/file.rs

```diff
@@ -18,6 +18,7 @@ pub fn parse(
     filename: impl AsRef<Path>,
     version: String,
     seeds_feature: bool,
+    relations_feature: bool,
```

Comment:
Should we combine this with the seeds_feature ?

Reply:
The nomenclature isn't accurate but the intention is the same.

Reply:
Up to you. You mean keep the same name? `seeds = true`

Reply:
Yes.